src/modules/search/searchApi.ts

```diff
@@ -47,23 +49,17 @@ export const searchResources = async (query: MultiSearchApiQuery) => {
   return resolveJsonOrRejectWithError(response);
 };
 
-export const searchDraft = async (query: DraftSearchQuery) => {
-  let response;
-  if (query) {
-    const types = query.types ? query.types.split(',') : [];
-    const realPageSize =
-      types.length > 1
-        ? Math.ceil((query['page-size'] || 5) / types.length)
-        : query['page-size'];
-    response = await fetchAuthorized(
-      `${baseUrl}/draft/?${queryString.stringify({
-        ...query,
-        'page-size': realPageSize,
-      })}`,
-    );
-    return resolveJsonOrRejectWithError(response);
-  }
-  response = await fetchAuthorized(`${baseUrl}/draft/`);
+export const searchAudio = async (query: MediaSearchQuery) => {
```

Comment:
Det finnes allerede egne moduler for image og audio. image har en searchImage som kan brukes, og audio kan oppdateres med tilsvarende. Ryddigere å ha desse i egne moduler i stedet for å legge dei inn her.

Reply:
Skal jeg også flytte håndtering av state og actions til modules/audio og modules/images?